lib/blas_magma.cu

```diff
@@ -1,8 +1,8 @@
 #include <blas_magma.h>
```

Comment:
Not sure if any of this MAGMA stuff is still relevant.  I think we don't use MAGMA at all anymore.  @alexstrel agree?